lua/mason-nvim-dap/mappings/configurations.lua

```diff
@@ -27,29 +27,33 @@ M.go = {
 	},
 }
 
+local BASHDB_DIR = require('mason-registry').get_package('bash-debug-adapter'):get_install_path()
+	.. '/extension/bashdb_dir'
+M.sh = {
```

Comment:
https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation

Maybe documenting it here would be good, so others have an idea